data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml

```diff
@@ -0,0 +1,957 @@
+ἐφαρμοζομένων ἐπ᾿ ἄλληλα καὶ τὰ κέντρα τῶν βαρέων
+ἐφαρμόζει ἐπ᾿ ἄλληλα πάντα γὰρ τὰ μέρη αὐτῶν πᾶσιν
+ἐφαρμόζει.</p>
+<p>Τῶν δὲ ἀνίσων, ὁμοίων δέ, τὰ κέντρα τῶν βαρέων
+<lb n="20"/> ὁμοίως ἔσται κείμενα.</p>
+<figure><graphic url=" http://heml.mta.ca/lace/sidebysideview2/12871653"/></figure>
+
+<pb n="168"/>
+<p>Νοείσθω δέ, ὡς ἐπὶ τῆς ὑποκειμένης καταγραφῆς, τὰ
+ΑΒΓ, &#9651;ΕΖ τρίγωνα ἄνισα καὶ ὅμοια, κέντρον δὲ βάρους
+τοῦ μὲν ΑΒΓ τὸ Η, τοῦ δὲ &#9651;ΕΖ τὸ Θ, καὶ ἐπεζεύχθωσαν
+αἱ ΑΗ, ΗΓ, ΒΗ, &#9651;Θ, ΘΕ, ΘΖ. Λέγω ὅτι εἰς ἴσα διαιροῦσιν
+<lb n="5"/> τὰς γωνίας αἱ ἀπὸ τῶν Η, Θ σημείων ἐπιζευχθεῖσαι.</p>
+<p>Γινέσθω γὰρ ὡς ἡ ΕΖ πρὸς ΒΓ, οὕτως ἡ ΕΘ πρὸς
+ΘΚ καὶ ἡ ΖΘ πρὸς ΘΛ καὶ ἡ &#9651;Θ πρὸς ΘΜ, καὶ ἐπεζεύχθωσαν
+αἱ ΜΚ, ΚΛ, ΛΜ· ἔσται δὴ ὅμοιον τὸ ΚΛΜ τρίγωνον τῷ
+&#9651;ΕΖ τριγώνῳ. Ἐπεὶ γάρ ἐστιν ὡς ἡ ΕΘ πρὸς ΘΚ, ἡ
+<lb n="10"/> ΘΖ πρὸς ΘΛ, παράλληλός ἐστιν ἡ ΕΖ τῇ ΚΛ· ὁμοίως
+δὴ καὶ ἡ ΜΚ τῇ &#9651;Ε καὶ ἡ ΛΜ τῇ &#9651;Ζ· ὅμοιον ἄρα τὸ &#9651;ΕΖ
+τρίγωνον τῷ ΚΛΜ τριγώνῳ. Ἔστιν ἄρα ὡς ἡ &#9651;Ε πρὸς ΜΚ,
+ἡ ΕΖ πρὸς ΚΛ καὶ ἡ &#9651;Ζ πρὸς ΜΛ. Ὑπόκειται δὲ διὰ
+τὴν ὁμοιότητα τῶν ΑΒΓ, &#9651;ΕΖ τριγώνων ὡς ἡ &#9651;Ε πρὸς
+<lb n="15"/> ΑΒ, ἡ ΕΖ πρὸς ΒΓ καὶ ἡ &#9651;Ζ πρὸς ΑΓ· ἴσαι ἄρα εἰσὶν
+αἱ ΑΒΓ ταῖς ΜΚΛ· ὥστε ἐφαρμόζει ἑκάστη ἐπὶ ἑκάστην.</p>
+<p>Ἴσον ἄρα καὶ ὅμοιόν ἐστι τὸ ΑΒΓ τρίγωνον τῷ ΚΜΛ
+τργώνῳ· ὥστε καὶ ἐφαρμόσει τὸ κέντρον τοῦ ΑΒΓ ἐπὶ
+τὸ τοῦ ΜΚΛ. Τοῦ δὲ Η ἐπὶ τὸ Θ ἐφαρμόζοντος καὶ τῶν
+<lb n="20"/> Α, Β, Γ ἐπὶ τὰ Μ, Κ, Λ ἐφαρμόσουσιν καὶ αἱ ΑΗ, ΒΗ,
+ΓΗ ἐπὶ τὰς ΜΘ, ΚΘ, ΛΘ καὶ ἴσας ποιήσουσιν γωνίας
+πρὸς τοῖς Μ, Κ, Λ ταῖς ἐν τῷ ΑΒΓ τριγώνῳ· ὥστε καὶ
+ἐν τῷ &#9651;ΕΖ· αἱ αὐταὶ γάρ εἰσιν εὐθεῖαι αἱ ἀπο τοῦ Θ ἐπί
+τε τὰ Μ, Κ, Λ καὶ ἐπὶ τὰ &#9651;, Ε, Ζ ἐπιζευγνύμεναι.
+<lb n="25"/> Παντὸς σχήματος, οὗ κα ἁ περίμετρος ἐπὶ τὰ αὐτὰ
+κοίλα ᾖ, τὸ κέντρον τοῦ βάρεος ἐντὸς εἶναι δεῖ τοῦ
+σχήματος | Τίνας καλεῖ τὰς ἐπὶ τὰ αὐτὰ κοίλας γραμμὰς
+εἴρηται ἡμῖν σαφῶς ἐν τοῖς προοιμίοις τοῦ Περὶ σφαίρας
+καὶ κυλίνδρου. Ἐπειδὴ δὲ τὸ σχῆμα τὸ ἐπὶ τὰ αὐτὰ
+<lb n="30"/> κοίλην ἔχον τὴν περίμετρον πάντα τὰ μέρη τοῦ ἐπιπέδου
+
+<pb n="169"/>
+ἐντὸς ἔχει καὶ τὰς γωνίας, δῆλον ὅτι καὶ τὸ κέντρον
+τοῦ βάρους ἐντὸς ἔχει τοῦ σχήματος· ἐπὶ γάρ τινων
+σχημάτων τὸ κέντρον τοῦ σχήματος ἐκτός ἐστι καὶ ἐπὶ
+<figure><graphic url=" http://heml.mta.ca/lace/sidebysideview2/12871653"/></figure>
+τῆε περιμέτρου. Ἐπὶ μὲν γὰρ τοῦ ΑΒΓ ἡμικυκλίου
+<lb n="5"/> κέντρον τοῦ σχήματός ἐστι τὸ Η, ἐπὶ δὲ τῆς &#9651;ΕΖ ὑπερβολῆς
+τὸ κέντρον τοῦ σχήματος ἐκτὸς ἐστιν, καθ᾿ ὃ αἱ διάμετροι
+συμπίπτουσιν ἀλλήλαις, ὡς ἔχει τὸ Θ· εἴρηται γὰρ
+ταῦτα ἐν τῷ δευτέρῳ βιβλίῳ τῶν Ἀπολλωνίου Κωνικῶν.
+Ὅμως δὲ καὶ ἐπὶ τοῦ ΑΒΓ σχήματος καὶ ἐπὶ τοῦ &#9651;ΕΖ
+<lb n="10"/> τὸ κέντρον τοῦ βάρους, ἀφ᾿ οὗ δηλονότι ἀρτώμενον
+τὸ σχῆμα παράλληλόν ἐστι τῷ ὁρίζοντι, ἐντός ἐστι τῆς
+περιμέτρου· εἰ γὰρ ἔσται ἐπὶ τῆς περιμέτρου ἢ ἐκτός,
+ῥέψει ἐπὶ θάτερα· ὅπερ οὐχ ὑπόκειται.</p>
+</div>
+
+<div type="textpart" subtype="section" n="2">
+<head>Εἰς τὸ δ΄.</head>
```

Comment:
Would it make sense to keep the greek numbering system here? It seems to be referring to something.

Reply:
Good catch. It is a commentary on tlg0552.tlg005, so I will add the URNs in a corresp attribute, as discussed in #201